crates/node/core/src/args/engine.rs

```diff
@@ -24,6 +27,10 @@ pub struct EngineArgs {
     #[arg(long = "engine.caching-and-prewarming")]
     pub caching_and_prewarming_enabled: bool,
 
+    /// Configure the size of cross-block cache
+    #[arg(long = "engine.cross-block-cache-size", default_value_t = DEFAULT_CROSS_BLOCK_CACHE_SIZE)]
```

Comment:
this should mention the unit,

for other caches we expect that the input is MB because handling bytes is a bit useless:

for example:

https://github.com/paradigmxyz/reth/blob/b955551f8b3b37b0ead6862e8c1bdae1e7ff69e9/crates/node/core/src/args/rpc_server.rs#L123-L125

Reply:
makes sense, done here https://github.com/paradigmxyz/reth/pull/14305/commits/0918a5b8f8205e0ba08f7590a421822b4fc038e8 PTAL